ingress/controller.go

```diff
@@ -10,30 +10,21 @@ import (
 	"fmt"
 
 	log "github.com/Sirupsen/logrus"
-	"github.com/sky-uk/feed/ingress/api"
+	"github.com/sky-uk/feed/api"
+	iApi "github.com/sky-uk/feed/ingress/api"
```

Comment:
Instead of `iApi`, can we rename `ingress/api` to `ingress/types`. Since it's internal to the ingress stuff, it makes sense to call it types here.